### PR TITLE
Minor clarification to "Renaming Foreman Server & Smart Proxy Servers"

### PR DESCRIPTION
**Pull request by maximiliankolb**
You can review commit by commit & we can squash when/before merging.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)

### CODE REVIEW

**Comment by adamlazik1** [comment]
May I suggest going all lowercase with already existing `smart-proxy-context` attribute?

**Comment by maximiliankolb** [comment]
@adamlazik1 Please have a look at the last commit. does this work for you? I am OK either way.

Generally speaking, I'd love to reduce the amount of attributes and esp. and way we "structure" attributes: some are "Project-ID", "ProjectID", project-id", "project-context", etc. Maybe a topic for another PR :)

File: guides/common/modules/proc_renaming-smart-proxy.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
-[id="Renaming_Smart_Proxy_{context}"]
-= Renaming {SmartProxyServerTitle}
+[id="renaming-{smart-proxy-context}-{context}"]
```
Comment (adamlazik1) [inline guides/common/modules/proc_renaming-smart-proxy.adoc:1]:
```suggestion
[id="renaming-{smart-proxy-context}_{context}"]
```

**Review by adamlazik1 — APPROVED** [review]
LGTM. Only one small change and then feel free to merge.

File: guides/common/modules/proc_renaming-smart-proxy.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_renaming-smart-proxy.adoc]: Nice catch, applied!

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked:
   dc3a7780f5..04f8c64340  3.10 -> 3.10
   97094b87e9..6a9ea59459  3.9 -> 3.9

**Pull request merged**